### ISSUE TITLE
`--pre` does not appear to work, at least not for @vue packages

### ISSUE THREAD

**Issue by RudeySH** [raineorshine/npm-check-updates#1013]
- [x] I have searched for [similar issues](https://github.com/raineorshine/npm-check-updates/issues)
- [x] I am using the latest version of `npm-check-updates`
- [x] I am using `node >= 12`

---------------------------

## Steps to Reproduce
- Create new folder
- `npm init --yes`
- `npm install @vue/cli-service@4.5.15 --save-dev`
- `ncu --upgrade --pre next`

## Current Behavior
`--pre` does not update [@vue/cli-service][1] from 4.5.15 to 5.0.0-rc.1

## Expected Behavior
`--pre` should update @vue/cli-service from 4.5.15 to 5.0.0-rc.1

[1]: https://www.npmjs.com/package/@vue/cli-service

**Comment by raineorshine** [comment]
It can be confusing! 

**TLDR;** Use `--target greatest`, not `--pre`.

Now the explanation.

First note that `--pre` only accepts `0` (false) or `1` (true). Any other value is treated as true. Default is false. So you should just use `--pre 1` to enable this option.

Prerelease filtering (`--pre`) and target tag (`--target`) are two separate options. The default tag is `latest`. In order to include other versions, you have to use `--target greatest` or `--target newest`. **`ncu --upgrade --pre next` does not have any upgrades for your because `4.5.15` is already the latest.** `--pre` is irrelevant since `4.5.15` is not a prerelease.

```sh
$ npm view  @vue/cli-service dist-tags                                                                        ✓
{ latest: '4.5.15', next: '5.0.0-rc.1' }
```

The most commonly used commands are:

- `ncu` says to upgrade to the `latest` tag, excluding prerelease versions.
- `ncu --target greatest` says to upgrade to the greatest possible number, including prerelease versions.

It is less common to use `--pre`:

- `ncu --pre 1` says to upgrade to the `latest` tag, including prerelease versions (which package authors should really not be publishing to `latest`).
- `ncu --pre 0 --target greatest` says to upgrade to the greatest possible number, excluding prerelease versions.

Hope that helps :)
